tips/TIP-0047/tip-0047.md

```diff
@@ -0,0 +1,119 @@
+individual output. This eases maintainability of this TIP as it only defines the principles of the storage deposit while
+other TIPs can build on top of these principles.
+
+# Motivation
+
+In a distributed ledger network, every participant, a so-called node, needs to keep track of the current ledger state.
+Since _Chrysalis Part 2_, the IOTA ledger state is based on the UTXO model, where every node keeps track of all the
+currently unspent outputs.
+
+Misusage by honest users or intentionally malicious behavior can lead to growing database and snapshot sizes and
+increasing computational costs (database lookups, balance calculations). Due to these increasing hardware requirements,
+the entry barrier to participate in the network becomes higher and less nodes would operate the network, hurting
+decentralization.
+
+Especially in a system like IOTA where token holders can issue blocks without fees, this is a serious issue, since an
+attacker can create a lot of damage with low effort. Other DLTs do not yet face this problem, as such an attack would be
+much more expensive due to the high transaction fees. However, in order to solve scalability issues more and more
+transactions need to be handled. Therefore, other DLT projects will also eventually run into the same dust limitations.
+
+The _Storage Deposit_ addresses this issue by creating an incentive to keep the ledger state small. In simple words: The
+larger an output's byte size, the more IOTAs need to be deposited in the output. This is however not a fee, because the
+deposited coins can be reclaimed by consuming the output in a new transaction.
+
+# Requirements
+
+- The maximum possible ledger database size must be limited to a reasonable and manageable size.
+- The dust protection must not depend on a global shared state of the ledger, so that transaction validation can happen
+  in parallel.
+- The dust protection should work for outputs with arbitrary data and size.
+- The ledger database size should be fairly allocated to users based on the scarce resource, IOTA coins.
+
+# Detailed Design
+
+Blocks including payloads, such as data or transaction payloads, are pruned by nodes after some time, but unspent
+transaction outputs must be kept until they are spent. Therefore the dust protection rules only apply to unspent
+outputs.
+
+This TIP specifies the following calculation of the storage deposit to simplify the calculation from TIP-19, using the
+_Storage Score Parameters_ of the [IOTA 2.0 Protocol Parameters](../TIP-0049/tip-0049.md):
+
+- Let the `Storage Score` of an output be the sum of:
+  - The `Storage Score Offset Output` of an output in storage, which is the sum of the `Offset Output Overhead` and
+    `Factor Data * Output Metadata Size` where `Offset Output Overhead` accounts for the overhead of storing any output
+    and `Output Metadata Size` is the size in bytes of the output's metadata.
```

Comment:
Is output metadata defined somewhere ?

Reply:
Apparently forgot to define it when writing the TIP. Just added it.